src/client/pages/Checkout/shared/CheckoutPageWrapper.tsx

```diff
@@ -87,7 +94,7 @@ export const CheckoutPageWrapper = ({
           </LogoLink>
         )}
         <BackButton onClick={handleClickBackButton} />
-        <InnerWrapper>{children}</InnerWrapper>
+        <InnerWrapper className={className}>{children}</InnerWrapper>
```

Comment:
why do we need the className here ?

Reply:
to be able to pass custome styles such as CheckoutPaymentWrapper